refactor: rebrand from Jellyseerr to Seerr across project

Gauvino opened the pull request:
#### Description

This PR continues the work from #2019 and #2111 by renaming additional "jellyseerr" references to "seerr" that were missed in previous refactors. Specifically:

- Renamed the documentation folder from `using-jellyseerr/advanced/` to `using-seerr/advanced/`
- Updated internal documentation references and links to use the new path structure
- Fixed broken links caused by the folder rename
- Addressed remaining "jellyseerr" references throughout the codebase

**Note:** The majority of remaining "jellyseerr" references are in translation files managed by Weblate.

#### To-Dos

- [X] Disclosed any use of AI (see our [policy](../CONTRIBUTING.md#ai-assistance-notice))
- [ ] Successful build `pnpm build`
- [ ] Translation keys `pnpm i18n:extract`
- [ ] Database migration (if required)

#### Issues Fixed or Closed

- Continues #2111
- Related to #2019
- Contributes to #926

Gauvino commented on gen-docs/blog/2025-09-29-introducing-seerr-blog.md:
```diff
@@ -0,0 +1,24 @@
+title: Welcome to the Seerr Blog
+description: The official Seerr blog for release notes, technical updates, and community news.
+slug: welcome
+authors: [fallenbagel, gauthier-th]
```
This will probably need to be updated also but i want to wait about seerr team to see what to do

Gauvino committed:
refactor: rebrand from Jellyseerr to Seerr across project

Updates repository references, Docker image paths, and documentation links to reflect the new Seerr branding. Replaces fallenbagel/jellyseerr references with seerr-team/seerr throughout README, Helm charts, and documentation files.

Removes legacy Jellyseerr blog post and image assets, replacing them with Seerr equivalents. Adds TODO comments for pending package renames on Arch Linux and NixOS distributions.

Maintains backwards compatibility by preserving old package references in version checking components until external package repositories complete their renames.

Gauvino commented on README.md:
```diff
@@ -7,7 +7,7 @@
 <a href="https://discord.gg/seerr"><img src="https://img.shields.io/discord/783137440809746482" alt="Discord"></a>
-<a href="https://hub.docker.com/r/fallenbagel/jellyseerr"><img src="https://img.shields.io/docker/pulls/fallenbagel/jellyseerr" alt="Docker pulls"></a>
+<a href="https://hub.docker.com/r/seerr-team/seerr"><img src="https://img.shields.io/docker/pulls/seerr-team/seerr" alt="Docker pulls"></a>
 <a href="http://translate.jellyseerr.dev/engage/jellyseerr/"><img src="http://translate.jellyseerr.dev/widget/jellyseerr/jellyseerr-frontend/svg-badge.svg" alt="Translation status" /></a>
```
This link also need to be updated with seerr domain

Gauvino committed:
Merge branch 'develop' into jellyseerr-to-seerr

M0NsTeRRR commented on README.md:
```diff
@@ -7,7 +7,7 @@
 <p align="center">
 <a href="https://discord.gg/seerr"><img src="https://img.shields.io/discord/783137440809746482" alt="Discord"></a>
-<a href="https://hub.docker.com/r/fallenbagel/jellyseerr"><img src="https://img.shields.io/docker/pulls/fallenbagel/jellyseerr" alt="Docker pulls"></a>
+<a href="https://hub.docker.com/r/seerr-team/seerr"><img src="https://img.shields.io/docker/pulls/seerr-team/seerr" alt="Docker pulls"></a>
```
It's `seerr/seerr` on dockerhub

Gauvino committed:
docs: update branding from Jellyseerr to Seerr and expand maintainer team

Updates repository URLs and translation service links to reflect the Seerr rebrand, replacing jellyseerr.dev references with seerr.dev and updating Docker Hub organization from seerr-team to seerr.

Adds six additional maintainers to the authors configuration (M0NsTeRRR, TheCatLady, sct, OwsleyJr, 0xSysR3ll, sudo-kraken) and includes them as co-authors on the welcome blog post.

Standardizes maintainer titles by removing "Co-" and "Developer" prefixes, establishing a consistent "Maintainer of Seerr" designation across all team members.

Gauvino commented:
Is it good for you now ? @M0NsTeRRR and @gauthier-th ?

Gauvino committed:
docs: update branch references and author profiles

Updates documentation to reference the main branch instead of develop branch for build instructions, reflecting the current repository structure.

Enhances author profiles with real names, professional titles, email addresses, and expanded role descriptions to provide better context about maintainers.

Gauvino commented on gen-docs/blog/authors.yml:
```diff
@@ -12,10 +12,72 @@ fallenbagel:
+  url: https://github.com/OwsleyJr
+  image_url: https://github.com/OwsleyJr.png
+  socials:
+    github: OwsleyJr
```
@OwsleyJr  I need you to confirm that the information is correct, thank you in advance!
Also, if you can give me your email so I can add it maybe?

Gauvino reviewed (commented):
Just made the authors.yml file with I think everyone in the seerr-team but need to make sure that all the information is correct, thanks in advance for the response!

Gauvino committed:
docs: update fallenbagel url

gauthier-th commented on gen-docs/blog/authors.yml:
I think we will edit it ourselves, we also have other modifications we want to do to the blog.

Gauvino replied:
It's done, but do you me to rebase this branch to only have one commit ?

Gauvino replied:
yes cause i can at the final if you want ?

Gauvino committed:
fix: revert blog change as suggested by reviwers

Reverts author list to original Jellyseerr maintainers and updates their titles and descriptions to correctly reference Jellyseerr instead of Seerr.

Removes additional authors that were added during the Seerr fork, keeping only the two core original maintainers (fallenbagel and gauthier-th) to accurately represent the project's heritage.

Gauvino committed:
Merge branch 'jellyseerr-to-seerr' of https://github.com/Gauvino/jellyseerr into jellyseerr-to-seerr

Gauvino committed:
docs: add GitHub social link for fallenbagel author

Adds GitHub profile reference to the blog author configuration to enable proper social media linking and improve author profile completeness.

gauthier-th reviewed (approved):
LGTM

gauthier-th reviewed (dismissed):
Please rename the `docs/using-jellyseerr/` folder: [here](https://github.com/seerr-team/seerr/tree/8bbffe3d35fe45ce879339d28e294d0ef40af6c2/docs/using-jellyseerr/advanced)

Gauvino committed:
fix(docs): move advanced folder from jellyseerr to seerr

0xSysR3ll commented:
> > > > Please rename the `docs/using-jellyseerr/` folder: [here](https://github.com/seerr-team/seerr/tree/8bbffe3d35fe45ce879339d28e294d0ef40af6c2/docs/using-jellyseerr/advanced)
> > > 
> > > 
> > > sorry that i missed that good now i think
> > > also what do you do about the weblate of seerr cause it's still jellyseerr
> > > and this branch could you rename it ?
> > > <img alt="image" width="1637" height="341" src="https://private-user-images.githubusercontent.com/68083474/515909981-545478b1-72d2-4c97-b552-b754d6081fd5.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjM1MDIyMDEsIm5iZiI6MTc2MzUwMTkwMSwicGF0aCI6Ii82ODA4MzQ3NC81MTU5MDk5ODEtNTQ1NDc4YjEtNzJkMi00Yzk3LWI1NTItYjc1NGQ2MDgxZmQ1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTExMTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMTE4VDIxMzgyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMzY2I2YWNiM2FkNjgyNWE2MjgyYjJhMjg2Y2E4MTc0OTkyMmUzM2MxNTJiZjAxYmRjNTM2MTAzYTliZGMzMWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.uOgnpvRqE1tiLahIDcrLDmOMge481mxPr5TSvRQwsVI">
> > 
> > 
> > Weblate translations will be taken care of, don't worry.
> > And the `legacy-jellyseerr` branch is for the old docs deployment that we are still using for now.
> 
> Ok so i don't have to do anything more, maybe delete this workflow or something, cause one workflow use this branch

Nope. That's all for this PR thanks :)
I you could just rebase so that we can merge please.

0xSysR3ll committed:
Merge branch 'develop' into jellyseerr-to-seerr
